src/Foundation.Find.Cms/Config/FacetEnums.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace Foundation.Find.Cms.Config
```

Comment:
Change namespace to Foundation.Find.Cms.Facets.Config

Reply:
Done